recipes/nifty/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set version = "v1.0.8" %}
+{% set checksum = "4a70eb106a926271cf86cae3e568c71b26ca40ecdfbfc337110f485410aa2876" %}
+
+package:
+  name: nifty
+  version: {{ version }}
+
+source:
+    url: https://github.com/DerThorsten/nifty/archive/{{ version }}.tar.gz
+    sha256: {{ checksum }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - ninja  # [win]
+    - make
+  host:
+    - python
+    - numpy
+    - xtensor >=0.21,<0.22
+    - xtensor-python >=0.24,<0.25
+    - vigra
+    - hdf5  # [not win]
+    - z5py >=2.0.8  # [not win]
+    # macos needs boost as rt dependency because of boost filesystem
+    - boost-cpp  # [osx]
+    # build dependencies needed for z5
+    - nlohmann_json  # [not win]
+    - blosc  # [not win]
+    - bzip2  # [not win]
+    - zlib  # [not win]
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+    - scikit-image
+    - boost-cpp
```

Comment:
@chrisburr boost-cpp is only necessary as run dependency for osx. Is it considered bad practice to pin it only for one operating system?

Reply:
Nope, it's just a typo when I fixed the macOS failure with 5309a28 😅